files/package.json

```diff
@@ -6,8 +6,8 @@
   "scripts": {},
   "devDependencies": {
     "concurrently": "^8.2.0",
-    "prettier": "^2.5.1",
-    "prettier-plugin-ember-template-tag": "^1.0.0"
+    "prettier": "^3.0.2",
```

Comment:
right now this was pretty busted for me
even the prettier vs code extension released a minor version rolling back the prettier 3 upgrade
https://github.com/prettier/prettier-vscode/blob/main/CHANGELOG.md#1010

Reply:
even after generating a new project from this branch? (with yarn or pnpm, not npm)

Also, why would vscode not use the project's prettier version?

it's working great for me over here

Reply:
what's busted? this is what fixes everything for me.

without this change, gts files are _deleted by prettier_

Reply:
I had to specify a `prettier.prettierPath` vs code config option for it to even work at all with version 3 didn't work otherwise and fallback would blow up since I didn't have a global version installed ...

Reply:
I don't have a global version installed. All my tools use local bins. does vscode not? :scream: